index.html

```diff
@@ -1787,6 +1876,172 @@ <h2>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        File Handler Items
+      </h2>
+      <p>
+        Each <dfn data-local-lt="file handler's">file handler</dfn> represents
+        a url in the scope of the application that can handle launches with
```

Comment:
Maybe just "handles launches" instead of "can handle launches" ? Less passive voice, I think.